cnf-tests/testsuites/e2esuite/gatekeeper/gatekeeper.go

```diff
@@ -694,6 +714,21 @@ var _ = Describe("gatekeeper", func() {
 			err = client.Update(context.Background(), gkConfig, &k8sClient.UpdateOptions{})
 			Expect(err).ToNot(HaveOccurred())
 
+			mutatinWebhookConfiguration := &admission.MutatingWebhookConfiguration{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "gatekeeper-mutating-webhook-configuration",
+				},
+			}
+			mwConfigKey, err := k8sClient.ObjectKeyFromObject(mutatinWebhookConfiguration)
+			Expect(err).ToNot(HaveOccurred())
+
+			Eventually(func() bool {
+				err := client.Get(context.Background(), mwConfigKey, mutatinWebhookConfiguration)
+				Expect(err).ToNot(HaveOccurred())
+				return mutatinWebhookConfiguration.Webhooks[0].NamespaceSelector != nil &&
```

Comment:
add a check for len of webhook is equal to 1 or you can have a panic here

Reply:
Added